process/mock/storerMock.go

```diff
@@ -57,22 +54,7 @@ func (sm *StorerMock) GetFromEpoch(key []byte, _ uint32) ([]byte, error) {
 	return sm.Get(key)
 }
 
-// GetBulkFromEpoch -
-func (sm *StorerMock) GetBulkFromEpoch(keys [][]byte, _ uint32) (map[string][]byte, error) {
-	retValue := map[string][]byte{}
-	for _, key := range keys {
-		value, err := sm.Get(key)
-		if err != nil {
-			continue
-		}
-		retValue[string(key)] = value
-	}
-
-	return retValue, nil
-}
-
-// SearchFirst -
-func (sm *StorerMock) SearchFirst(_ []byte) ([]byte, error) {
+func (sm *StorerMock) GetBulkFromEpoch(keys [][]byte, _ uint32) ([]common.KeyValuePair, error) {
```

Comment:
why this change on the mock?
Missing comment

Reply:
Added comment, left not implemented (function not used).

Reply:
ok